### PR TITLE
Task/update from upstream

### PR DESCRIPTION
**Pull request by mariopepe**

### CODE REVIEW

File: intercom_flutter_platform_interface/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,17 @@
...
 - Updated `company` parameter inside `updateUser` method.
+=======
+# 1.2.2
+
+- Added method `displayMessages`.
+
+# 1.2.1
+
+- Updated dependencies
+>>>>>>> origin/master
```
Comment (lifedemons) [inline intercom_flutter_platform_interface/CHANGELOG.md]: @mariopepe this doesn't look good, these are the leftovers from the merge, so we need in ideal case to remerge it

File: intercom_flutter_platform_interface/pubspec.yaml [hunk 2]
```diff
@@ -1,17 +1,21 @@
 name: intercom_flutter_platform_interface
 description: A common platform interface for the intercom_flutter plugin.
+<<<<<<< HEAD
```
Comment (lifedemons) [inline intercom_flutter_platform_interface/pubspec.yaml]: same here

File: intercom_flutter_web/CHANGELOG.md [hunk 3]
```diff
@@ -1,8 +1,18 @@
 # Changelog
 
+<<<<<<< HEAD
 # 0.3.0
 - Fixed `company` attribute mapping.
 - Updated dependency `intercom_flutter_platform_interface: ^1.3.0`
+=======
```
Comment (lifedemons) [inline intercom_flutter_web/CHANGELOG.md]: and here

**Review by lifedemons — CHANGES_REQUESTED** [review]

File: intercom_flutter_platform_interface/CHANGELOG.md (hunk 1 above)
Comment (mariopepe) [reply intercom_flutter_platform_interface/CHANGELOG.md]: I already solved them yesterday, but forgot to push.

Actually i pushed, but wrongly, the right command was the one you gave me:

git push quandoo quandoo_master:task/update_from_upstream

**Review by lifedemons — APPROVED** [review]

**Pull request merged**